pkg/yurtctl/cmd/convert/convert.go

```diff
@@ -377,6 +416,84 @@ func (co *ConvertOptions) RunConvert() (err error) {
 	return
 }
 
+func deployYurtAppManager(
+	client *kubernetes.Clientset,
+	cloudNodes []string,
+	yurtappmanagerImage string,
+	yurtAppManagerClient dynamic.Interface) error {
+
+	// 1.create the YurtAppManagerCustomResourceDefinition
+	// 1.1 nodepool
+	if err := kubeutil.CreateCRDFromYaml(client, yurtAppManagerClient, "",[]byte(constants.YurtAppManagerNodePool)); err != nil {
```

Comment:
It seems like the func `CreateCRDFromYaml` is to create a cr. However, it's used to register crd here.

Reply:
register crd